backend/private-graph/graph/schema.graphqls

```diff
@@ -577,20 +592,36 @@ type Log {
 	serviceName: String
 }
 
-type LogEdge {
+type LogEdge implements Edge {
 	cursor: String!
 	node: Log!
 }
 
-type PageInfo {
-	hasNextPage: Boolean!
-	hasPreviousPage: Boolean!
-	startCursor: String!
-	endCursor: String!
+type LogConnection implements Connection {
+	edges: [LogEdge!]!
+	pageInfo: PageInfo!
 }
 
-type LogsConnection {
-	edges: [LogEdge!]!
+type ErrorObjectNodeSession {
+	secureID: String!
+	userProperties: String!
+	appVersion: String
+}
+
+type ErrorObjectNode {
+	id: ID!
+	createdAt: Timestamp!
+	event: String!
+	session: ErrorObjectNodeSession!
+}
+
+type ErrorObjectEdge implements Edge {
+	cursor: String!
+	node: ErrorObjectNode!
```

Comment:
I would have liked to have used `ErrorObject` here but there seems to be something [broken](https://highlightcorp.slack.com/archives/C02CJANPHQS/p1686078353817089) with gqlgen.